packages/stylelint-config/package.json

```
@@ -36,7 +36,7 @@
 		"stylelint-scss": "^3.17.2"
 	},
 	"peerDependencies": {
-		"stylelint": "^13.7.0"
+		"stylelint": ">=13.7.0"
```

I think we should be strict here and not declare support for all upcoming versions.

```suggestion
		"stylelint": "^13.7.0 || ^14"
```

Similar is done for ESLint
https://github.com/WordPress/gutenberg/blob/a4da33837e1638db12b21ea83cbeebd7a9289e9b/packages/eslint-plugin/package.json#L52

---

Amazing, did not think about the 'or' operator. Danke!